code/molecular_phenotypes/calling/RNA_calling.ipynb

```diff
@@ -735,7 +735,81 @@
   {
    "cell_type": "code",
    "execution_count": 8,
-   "id": "fancy-thermal",
+   "id": "improving-mississippi",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "[rnaseqc_call_3]\n",
+    "parameter: reference_fasta = path \n",
+    "parameter: ref_flat = path\n",
+    "parameter: riboIntervals = path\n",
+    "# Maximum distance in pixels between two duplicate clusters in order to consider them optical duplicates.\n",
+    "# The default is appropriate for unpatterned versions of the Illumina platform. \n",
+    "# For the patterned flowcell models (HiSeq X), 2500 is more appropriate.\n",
+    "parameter: optical_distance = 100\n",
+    "input: group_by = 2, group_with = 'sample_id'2356\n",
+    "output: f'{cwd}/{_sample_id}.Aligned.sortedByCoord.out.md.bam',\n",
+    "        f'{cwd}/{_sample_id}.Aligned.sortedByCoord.metrics',\n",
+    "        f'{cwd}/{_sample_id}.Aligned.sortedByCoord.duplicate_metrics',\n",
+    "        f'{cwd}/{_sample_id}.Aligned.sortedByCoord.rna_metrics',\n",
+    "bash: container=container, expand= \"${ }\", stderr = f'{_output:n}.stderr', stdout = f'{_output:n}.stdout'\n",
+    "\n",
+    "        java -Xmx8g -jar /opt/picard-tools/picard.jar MarkDuplicates \\\n",
```

Comment:
Here you are using 8g memory for JAVA , hard coded. This should be fixed to using user specified input? 

Also, it should be a `task: `.